shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -149,6 +156,14 @@ @interface FlutterTextInputView : UIView <UITextInput>
 @property(nonatomic, strong) UITextRange* markedTextRange;
 @property(nonatomic, copy) NSDictionary* markedTextStyle;
 @property(nonatomic, assign) id<UITextInputDelegate> inputDelegate;
+/**
+ This file is to make ios 10 or higher device using autofill
+ 
+ Reference to  github.com/flutter/flutter/issues/33040
+ **/
+if(@available(10.0,0))
+@property(nonatomic) UITextContentType textContentType;
```

Comment:
Is this compiling and working properly?  I was recently having trouble with getting the syntax right for specifying the version of iOS like this in https://github.com/flutter/engine/pull/13863.  I thought it should be:

```dart
@property(nonatomic) UITextContentType textContentType API_AVAILABLE(ios(10.0));
```

Reply:
I will check it now. Sorry for none testing.